components/upload/overview.md

```diff
@@ -172,7 +172,60 @@ The <a href="https://www.telerik.com/blazor-ui/upload" target="_blank">Blazor Up
 
 * [Validation]({%slug upload-validation%})
 
+## Methods
 
+The Upload methods are accesible through its reference.
+
+* `ClearFiles` - Clears all files from the list, both uploaded and in queue.
+  
+* `UploadFiles` - Uploads all valid selected files. Fires the [OnUpload]({%slug upload-events%}#onupload) event.
+
+* `OpenFileSelectAsync` - Triggers the browser's file select dialog.
+
+>caption Get a reference to the Upload and use its methods. The example uses the same controller as the above demo.
+
+````CSHTML 
+@* This example showcases the use of the methods. *@
```

Comment:
We need to add the controller configuration to the snippet. Please, assure that the snippet is runnable in Telerik Repl. If it  is not and should not be - add the skip repl div (might need to sync with the support engineers how to do this, but if we can have it runnable, we should)